bedrock/entity/send_message.py

```diff
@@ -0,0 +1,30 @@
+import asyncio
+import sys
+
+from temporalio.client import Client
+from workflows import BedrockParams, EntityBedrockWorkflow
+
+
+async def main(prompt):
+    # Create client connected to server at the given address
+    client = await Client.connect("localhost:7233")
+
+    workflow_id = "entity-bedrock-workflow"
+
+    # Sends a signal to the workflow (and starts it if needed)
+    await client.start_workflow(
+        EntityBedrockWorkflow.run,
+        arg=BedrockParams(None, None),
```

Comment:
```suggestion
        BedrockParams(None, None),
```

Shouldn't need a kwarg for this